### PR TITLE
OpenXR - Refactor

### PR DESCRIPTION
**Pull request by lvonasek**
* Move PPSSPP specific code into PPSSPPVR.cpp
* Refactor VRMath (redundant structures, methods, etc.)
* VRMath contains now only methods working with OpenXR structures

This is an initial step to move camera adjust and stereo view calculation out of VR.

### CODE REVIEW

File: Common/VR/PPSSPPVR.cpp
Comment (unknownbrackets) [inline Common/VR/PPSSPPVR.cpp:474]: `Lin::Matrix4x4` has a method that would do this and then you could simply do `M.wx *= -1.0f;` etc. before the memcpy().  Just noting.

-[Unknown]

File: Common/VR/PPSSPPVR.cpp
Comment (unknownbrackets) [inline Common/VR/PPSSPPVR.cpp]: I realize cassert wasn't here, but personally I like using defensive asserts like this in case I forget to update a condition to add a new matrix case or etc. later.

-[Unknown]

**Review by unknownbrackets — APPROVED** [review]

File: Common/VR/PPSSPPVR.cpp
Comment (lvonasek) [reply Common/VR/PPSSPPVR.cpp:474]: Thanks, I see there method for ortho but not for perspective. I will keep it as it is as I would rearrange the values so much that it wouldn't make the code cleaner.

**Review by hrydgard — APPROVED** [review]
Looks good to me!

**Comment by lvonasek** [comment]
Thanks, then let's merge it (I do not have rights to do that).

**Comment by hrydgard** [comment]
I'll merge this after #16355 is merged (and passes)

**Pull request merged**
